internal/kafka/command_topic_consume.go

```diff
@@ -44,6 +44,7 @@ func (c *command) newConsumeCommand() *cobra.Command {
 	pcmd.AddKeyFormatFlag(cmd)
 	pcmd.AddValueFormatFlag(cmd)
 	cmd.Flags().Bool("print-key", false, "Print key of the message.")
+	cmd.Flags().Bool("print-offset", false, "Print offset of the message.")
```

Comment:
This also prints the partition number, right?

Reply:
yes. updated the description